L/libbacnet_stack/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libbacnet_stack"
+version = v"1.4.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/bacnet-stack/bacnet-stack.git", "c111154993cce5dc0e3edd10c602bce51e2d2d61")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cmake -B build -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON bacnet-stack
+cmake --build build --parallel 4
```

Comment:
```suggestion
cmake --build build --parallel ${nproc}
```
We tell the build how many processes to use through this variable.